README.md

```diff
@@ -1403,6 +1403,31 @@ However, when we try to call `makeSound()` on an instance of the Animal class `(
 
 ---
 
+###### 42. Default Import Export (Level: Medium)
+
+In JavaScript, when importing a default export from a module, which syntax correctly assigns an alias "myAlias" to the default import?
+
+- A: import { myAlias as default } from 'module';
+- B: import default as myAlias from 'module';
+- C: import myAlias as default from 'module';
+- D: import { default as myAlias } from 'module';
+
+
+
+<details><summary><b>Answer</b></summary>
+<p>
+
+#### Answer: D
+
+The correct answer is D.
```

Comment:
Update explanation to support the changes above to cover both ways of adding alias. Please also mention that option B is the recommended way of adding alias

Here is a SO post for you if you want to read more: https://stackoverflow.com/questions/39282253/how-can-i-alias-a-default-import-in-javascript